bot/plotting/random_plot_generator.py

```diff
@@ -3,40 +3,50 @@
 import logging
 from plotting.plot_graph import plot_graph
 from models.model_solver import model_solver
-from utils.chemistry_generator import chemistry_generator
+from utils.parameter_value_generator import parameter_value_generator
 from experiment.experiment_generator import experiment_generator
 from experiment.experiment_solver import experiment_solver
 from plotting.summary_variables import generate_summary_variables
+from plotting.comparison_generator import comparison_generator
 
 
 def random_plot_generator(
     return_dict,
     testing=False,
     provided_choice=None,
-    provided_number_of_comp=None,
-    plot_summary_variables=True,
+    provided_chemistry=None,
     provided_degradation=True,
 ):
     """
     Generates a random plot.
     Parameters:
         return_dict: dict
+            A shared dictionary in which all the return values are stored.
         testing: bool
```

Comment:
Just a suggestion, it might be cleaner to pass an `options` dict which is by defualt `{"choice": None, "number of comparisons": None, "chemistry": None}` then you can just check e.g. 
```python3
if options[chemistry] is None:
    options[chemistry] = random.choice(chemistries)
````

And you can do the same in the other scripts.

Reply:
It probably shouldn't be called options to avoid clashing with the model options, or you could called the options for the model `model_options`

Reply:
Yes that will make it more readable, I'll make the changes